codefresh/README.md

```diff
@@ -170,6 +180,17 @@ ingress-nginx:
       --timeout 15m
   ```
 
+## Post-installation steps
+
+- Disable auto-creation of MongoDB indexes (or remove these values from `.Values`):
```

Comment:
Let's extend a bit: “Before moving to production, MongoDB indexes auto-creation must be disabled by either setting ... or removing ...”